packages/forms/resources/views/components/checkbox-list.blade.php

```diff
@@ -202,7 +202,11 @@ class="fi-fo-checkbox-list-option-label flex gap-x-3"
                             <span
                                 class="fi-fo-checkbox-list-option-label overflow-hidden break-words font-medium text-gray-950 dark:text-white"
                             >
-                                {{ $label }}
+                                @if ($isHtmlAllowed)
```

Comment:
This should be `$isHtmlAllowed()`, this is currently always passing and allowing HTML in all cases, a security issue